### ISSUE TITLE
Validate storage capacity of pumped hydro stations in Europe

### ISSUE THREAD

**Issue by timtroendle** [timtroendle/euro-calliope#12]
The storage capacity of pumped hydro stations in Europe in the JRC hydro database is more than 10 TWh, which is an order of magnitude above other estimations.

Currently, we [scale capacities](https://github.com/timtroendle/euro-calliope/blob/684d25242ded36607b9cd07d021ec2110270e35e/src/hydro.py#L71) to match (Geth et al., 2015) data. They estimated 1.3 TWh in all Europe.

Where does this deviation come from?

**Comment by brynpickering** [comment]
So maybe I'm reading it incorrectly, but [Swiss data on reservoir fill levels](https://www.bfe.admin.ch/bfe/en/home/supply/statistics-and-geodata/energy-statistics/electricity-statistics.html) would suggest that they have ~8.5TWh of storage capacity for pumped and dammed RoR

**Comment by timtroendle** [comment]
Right, but most of this is dammed RoR, no?

**Comment by brynpickering** [comment]
Yep, but impossible to tell how much since everything is interconnected in cascades, where the storage availability of a dammed RoR is dependent on the use of an upstream pumped hydro plant, or vice versa... it would be even better to model all these cascades in the model ;)

**Comment by timtroendle** [comment]
Yeah, that makes it complicated. But actually, where we _do know_ about these things, modelling it could be an option. Like in Switzerland.

**Comment by brynpickering** [comment]
Yep, and [here](https://github.com/fonew/hydropower) is the data in the Swiss context. It's quite a complex representation of the Swiss cascade system (from an energy system point of view, at least), and I'm not sure how one would simplify it to our needs.
